### ISSUE TITLE
Support onDidAppear etc

### ISSUE THREAD

**Issue by helje5** [swiftwebui/SwiftWebUI#13]
TBD how and when those would get invoked, but it would be useful, e.g. in tab containers.

Maybe those are the awake/sleep of WOComponent's?

**Comment by curveddesign** [comment]
Could onDidAppear() or onAppear() be called the first time the web page loads?

**Comment by helje5** [comment]
No, one might navigate within the SPA. Probably not worth doing those.

**Comment by curveddesign** [comment]
I want to execute a swift function when the page loads?   I could create a flag environment variable so that is only runs the first time or if the data loaded is not present.   Is there another way I could do that?

**Comment by helje5** [comment]
I don't think so. I mean you can put stuff into the View's `init` or `body`, but those can be called multiple times. Though you can put things into the environment or state.

**Comment by curveddesign** [comment]
Yes, that is why I was looking to use onAppear(), I have used that with an environment flag in the real SwiftUI!  😃
